core/cautils/customerloader.go

```diff
@@ -63,6 +67,44 @@ func (co *ConfigObj) Config() []byte {
 	return []byte{}
 }
 
+func (co *ConfigObj) UpdateEmptyFields(inCO *ConfigObj) error {
```

Comment:
```suggestion
func (co *ConfigObj) updateEmptyFields(inCO *ConfigObj) error {
```
Should be a private function